projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -292,6 +292,15 @@ private function build_task( $task ) {
 		$built_task['badge_text']   = $this->load_value_from_callback( $task, 'badge_text_callback' );
 		$built_task['isLaunchTask'] = isset( $task['isLaunchTask'] ) ? $task['isLaunchTask'] : false;
 
+		if ( isset( $task['target_repetitions'] ) ) {
+			$built_task['target_repetitions'] = $task['target_repetitions'];
+			$repetition_count                 = $this->load_repetition_count( $task );
+			$built_task['repetition_count']   = $repetition_count;
```

Comment:
Now that I see this written out, I don't think we need the variable.
```suggestion
			$built_task['repetition_count']   = $this->load_repetition_count( $task );
```